README.md

```diff
@@ -145,10 +145,13 @@ $ sam deploy \
 Build the layer by:
 
 1. Installing a Docker environment
-1. Running `make`
+2. Running `make`
 
 This will launch a Docker container that will build php71.zip.
 
+Example of installing other php modules:
+https://medium.com/@narwy/stackery-php-lambda-layer-add-mysql-1e49f3f3160a
```

Comment:
I would rather expand the documentation in this project than link to somewhere else for helpful instructions. Maybe you could help make the documentation in the extensions section above more clear instead?